src/GafferML/Inference.cpp

```diff
@@ -60,6 +60,31 @@ using namespace GafferML;
 namespace
 {
 
+std::vector<std::string> customOpLibraryPaths()
+{
+	std::vector<std::string> path_list;
+
+	const char* env_var = std::getenv( "ONNX_CUSTOM_OP_LIBRARY" );
```

Comment:
Since it's GafferML that handles this env-var, and it is a list of multiple libraries, should we call it `GAFFERML_CUSTOM_OPS_LIBRARIES`?

Reply:
addressed here 6f5be1b85acd1b28627a9ebdd37ee840820071c9